flutter/lib/src/integrations/screenshot_integration.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import 'package:flutter/scheduler.dart';
+import 'package:sentry/sentry.dart';
+import '../screenshot/screenshot_attachment_processor.dart';
+import '../sentry_flutter_options.dart';
+
+class ScreenshotIntegration implements Integration<SentryFlutterOptions> {
+  SentryFlutterOptions? _options;
+
+  @override
+  FutureOr<void> call(Hub hub, SentryFlutterOptions options) {
```

Comment:
Ok, so users woudl need to add `SentryWidget` and opt in into screenshots by setting the boolean to true on options.

Reply:
No, for now, they need to do both, later at some point, we will make `SentryWidget` only and the boolean, because the screenshot widget will be done automatically.
Relates to https://github.com/getsentry/sentry-dart/pull/1088#discussion_r1004368568